sudokuSolver/sudokuSolver/sudokuSolverFunctions.cpp

```diff
@@ -0,0 +1,174 @@
+#include "sudokuSolverFunctions.h"
+
+int findMissing(int a[], int b[], int n, int m)
+{
+    for (int i = 0; i < n; i++)
+    {
+        int j;
+        int counter=0;
+        for (j = 0; j < m; j++)
+        {
+            if (a[i] == b[j])
+            {
+                counter++;
+            }
+        }
+        if(counter==0)
+        {
+            return a[i];
+        }
+    }
+}
+
+bool zeroValidator(int a[])
+{
+    for (int i=0; i<9; i++)
```

Comment:
avoid using magic numbers. You can always define some variable using #define and use it instead of number 9